packages/atlas/src/views/global/YppLandingView/YppLandingView.styles.ts

```diff
@@ -0,0 +1,50 @@
+import { css } from '@emotion/react'
+import styled from '@emotion/styled'
+
+import { media, sizes, zIndex } from '@/styles'
+
+export const Wrapper = styled.div`
+  padding-top: ${sizes(24)};
+  text-align: center;
+`
+
+export const HeroWrapper = styled.div`
+  position: relative;
+`
+
+const commonHeroImagesStyles = css`
+  position: absolute;
+  left: 50%;
+`
+
+export const FrontImage = styled.img<{ parallaxPosition: number }>`
+  ${commonHeroImagesStyles};
+
+  z-index: ${zIndex.nearOverlay};
+  transform: translate(-50%, -${({ parallaxPosition }) => parallaxPosition}px);
+  width: 100%;
+  will-change: transform;
+
+  ${media.md} {
+    max-width: 888px;
+  }
+
+  ${media.lg} {
+    max-width: 1152px;
+  }
+`
+
+export const BackImage = styled.img`
+  ${commonHeroImagesStyles};
+
+  transform: translateX(-50%);
+  width: 84%;
```

Comment:
I don't like how we position all of these images. How did you measure that? This 84%? I'm a little concerned, that it could be hard to manage in the future. This could be written much simpler. 

I would just, use the approach presented by Adam in Figma. Just export images with transparent parts, set their widths to 100%, and paste them.  We wouldn't need to set max-width separately for both of these images. WDYT? What are the benefits of your approach?

Reply:
No, no you are 100% right. I just forgot to export it after our discussion